### ISSUE TITLE
Borderlands no longer going ingame

### ISSUE THREAD

**Issue by Shoegzer** [RPCS3/rpcs3#5332]
Borderlands (NPUB30413) does not go ingame on this test system as of 0.0.5-7494-fada8b65 (though it's been like that for quite a while) Specifically, the game boots until the "2K" logo is finished; then it presents a screen with the upside-down "V" and circle around it in the middle of the screen, and seems to hang there even though the screen itself animates (pictures behind the "V" slide around).

I tried clearing all caches, deleting save game and trophy data, deleting configuration data, finally deleting the ~/.config/rpcs3 directory and starting from scratch, and even reinstalling the game at one point but nothing seemed to work.   

This was tested on a Linux x64 system with kernel 4.15, i7-7820HK CPU and an NVIDIA GTX 1070. Interestingly, [this video](https://www.youtube.com/watch?v=DM4G-lRgz40) shows the game running on a similarly recent build without an issue - though I assume that system is Windows. I can't conclude whether this is a "Linux thing" yet though, but hopefully someone else can.

Note also: I filed [this report](https://github.com/RPCS3/rpcs3/issues/4462) months ago, with the same symptoms, but that was cleared up with the SPU ASMJIT rewrite.  It worked for a month or so after that, then stopped working again.

Log file is here: [RPCS3.log](https://github.com/RPCS3/rpcs3/files/2572596/RPCS3.log)

**Comment by Shoegzer** [comment]
Good question.  I'll need to do a bisect for that.  Since it worked after the SPU ASMJIT rewrite, I'll have to use that as a starting point, though it will take a while to do that so please bear with me.  I also wanted to post this issue in case others encountered it too and could confirm what I'm seeing, or found a workaround that I may have missed.

**Comment by Deminating** [comment]
Testing on W10, RPCS3 v0.0.5-7497-0044eb44 it goes ingame just fine for me.
So I can confirm at least that it works on Windows.

i7 8700k, GTX 1070. Attaching log just in case it's remotely helpful
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/2589136/RPCS3.log.gz)

**Comment by Shoegzer** [comment]
@Deminating thanks, at least it narrows things down a bit, since we have the same graphics card - now we just need one other person with Linux to confirm it's broken there.  

@AniLeo I tried a bisect but it won't compile that far back due to what I'm guessing are incompatible Qt dev libraries from looking at the make log.  However I do recall that it was working as of bd1d4de422cd3fbf6839e10012cc5d7873f4f114 (again right after the PPU ASMJIT rewrite) before I (re)encountered the issue as noted above shortly thereafter.  

@Nekotekina realizing you're likely busy, any sparce cycles you might have to look into this one would be appreciated.

**Comment by Shoegzer** [comment]
Update: I just check against commit 65ca934452726ab8a667fe6a9c903489ddb67da9 and it works again (!), so i'm closing this issue, though it still would have been nice to know what actually causes this lest it happen again.  For reference, the issue must have been fixed since the last time I checked it, which was again fada8b65 about a week ago.  

Thanks to all involved in this thread for your help and info.
